AoA/scan_strategies/analyze_schedule.py

```diff
@@ -0,0 +1,110 @@
+from glob import glob
+import os
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+fig = plt.figure(figsize=[18, 12])
+nrow, ncol = 2, 2
+ax1 = fig.add_subplot(nrow, ncol, 1)
+ax2 = fig.add_subplot(nrow, ncol, 2)
+ax3 = fig.add_subplot(nrow, ncol, 3)
+
+colors = ["tab:blue", "tab:orange", "tab:green", "tab:pink"]
+
+if len(sys.argv) == 1:
+    print("Usage: {sys.argv[0]} <input directory>")
+indir = sys.argv[1]
+fnames = sorted(glob(indir + "/*txt"))
+
+for fname, color in zip(fnames, colors):
+    print("\n{}".format(fname))
+
+    with open(fname, "r") as fin:
+        fin.readline()
+        header = fin.readline()
+    site, telescope, lat, lon, alt = header.split()
+    if np.abs(float(lat)) > 85:
+        pole_site = True
+    else:
+        pole_site = False
+
+    schedule = np.genfromtxt(fname, skip_header=3).T
+
+    arr = np.genfromtxt(
+        fname,
+        skip_header=3,
+        usecols=[4, 5, 7, 8, 9, 10],
+        dtype="f8,f8,S30,f8,f8,f8",
+        names=["starts", "stops", "names", "az_mins", "az_maxs", "elevations"],
+    )
```

Comment:
This seems a lot nicer than the ground schedule reading in toast (https://github.com/hpc4cmb/toast/blob/51f896b1867ef13c9e8c1476e0c1cfae24ab6fac/src/toast/schedule.py#L271).  Perhaps eventually we could unify ground schedule I/O into the `GroundSchedule` class like the `SatelliteSchedule`.  Then we could read / write them any time we need.